client/webpack.client.rails.build.config.js

```diff
@@ -18,14 +18,16 @@ config.module.loaders.push(
     test: /\.css$/,
     loader: ExtractTextPlugin.extract(
       'style',
-      'css?minimize&modules&importLoaders=1&localIdentName=[name]__[local]__[hash:base64:5]'
+      'css?minimize&modules&importLoaders=1&localIdentName=[name]__[local]__[hash:base64:5]' +
+      '!postcss'
     ),
   },
   {
     test: /\.scss$/,
     loader: ExtractTextPlugin.extract(
       'style',
       'css?minimize&modules&importLoaders=3&localIdentName=[name]__[local]__[hash:base64:5]' +
+      '!postcss' +
       '!sass' +
       '!sass-resources'
```

Comment:
Also, I think the `webpack.server.rails.build.config.js` file also needs the postcss loader?